scenarios/crowdsecurity/pfsense-gui-bf.yaml

```diff
@@ -0,0 +1,14 @@
+# pfsense web auth bruteforce
```

Comment:
You need to add MITRE labels to your scenarios, you can most likely copy and paste from opnsense one